script.js

```diff
@@ -0,0 +1,428 @@
+	let mealSelector = document.getElementById('meal');
+    
+    mealSelector.addEventListener('change', function(){
+    	// find the value of the selected option
+    	let mealSelected = mealSelector.options[mealSelector.selectedIndex].value;
+		let makeVis = document.getElementById('mealOptionsHolder');
+    	// make options visible based off of selected value
+    	if (mealSelected != ''){
+    		
+    		let makeVisBreak = document.getElementById('breakfastOptions');
+    		let makeVisLunch = document.getElementById('lunchOptions');
+    		let makeVisDinner = document.getElementById('dinnerOptions');
+    		makeVis.className = 'visible';
+    		// would probably also make some code to remove the radio button selected if meal is changed
+    		
+    		if (mealSelected == 'breakfast') {
+    			makeVisBreak.className = 'visible';
+    			makeVisLunch.className = 'hidden';
+    			makeVisDinner.className = 'hidden';    			
+    		}
+    		else if (mealSelected == 'lunch') {
+    			makeVisBreak.className = 'hidden';
+    			makeVisLunch.className = 'visible';
+    			makeVisDinner.className = 'hidden';	
+    		}
+    		else if (mealSelected == 'dinner') {
+    			makeVisBreak.className = 'hidden';
+    			makeVisLunch.className = 'hidden';
+    			makeVisDinner.className = 'visible';
+    		}
+    		
+    		
+    	}
+    	else {
+    			makeVis.className = 'hidden';
+    			
+    	}
+    	
+    	
+
+    })
+
+    let fName = document.getElementById('firstName');
+    	fName.addEventListener('blur', function (){
+    		
+
+    		//Check for properly entered format
+    		
+    		let fNameCheck = this.value;
+    		let fName_f = '';
+    		//check to make sure a string has been entered that is not a number
+
+    		if (fNameCheck != '' || fNameCheck.isNaN){
+    			
+    			//Run though all letters and capitalize or "lowerize" accordingly
+    			for (let x = 0; x < fName.value.length; x++){
+    				if (x== 0){
+    					fName_f += fName.value[x].toUpperCase();
+    				}
+    				else {
+    					fName_f += fName.value[x].toLowerCase();
+    				}
+    			}
+    		}
+    		//alert and won't let you continue
+    		else{errorMessage = 'First name cannot be blank or a number';
+    			fName.focus();
+    		}
+    		// put corrected value in the form
+    		fName.value = fName_f;
+
+    		
+    })
+
+    let lName = document.getElementById('lastName');
+    lName.addEventListener('blur', function (){
+    		
+
+    		//Check for properly entered format
+    		
+    		let lNameCheck = this.value;
+    		let lName_f = '';
+    		
+    		//check to make sure a string has been entered that is not a number
+
+    		if (lNameCheck != '' || lNameCheck.isNaN){
+    			//Run though all letters and capitalize or "lowerize" accordingly
+   			
+    			for (let x = 0; x < lName.value.length; x++){
+    				if (x== 0){
+    					lName_f += lName.value[x].toUpperCase();
+    				}
+    				else {
+    					lName_f += lName.value[x].toLowerCase();
+    				}
```

Comment:
I notice this block of code is in both the fname and lname listeners...could you make it a function?